Packs/Elasticsearch/Integrations/Elasticsearch_v2/Elasticsearch_v2.py

```diff
@@ -289,6 +289,23 @@ def test_general_query(es):
         return_error("Fetch incidents test failed.\nError message: {}.".format(str(e).split(',')[2][2:-1]))
 
 
+def test_general_query(es):
+    """Test executing query to all available indexes.
+
+    Args:
+        es(Elasticsearch): an Elasticsearch object to which we run the test.
+    """
+    try:
+        query = QueryString(query='*')
+        search = Search(using=es, index='*').query(query)[0:1]
+        response = search.execute().to_dict()
+        _, total_results = get_total_results(response)
+
+    except NotFoundError as e:
+        return_error("Failed executing general search command - please check the Server URL and port number "
+                     "and the supplied credentials.\nError message: {}.".format(str(e).split(',')[2][2:-1]))
```

Comment:
whats with the error message here? seems like its prone to bugs to index error

Reply:
Honestly, I do not recall how the error looks like but this format has been in the integration since its inception a year ago with no issues to it. I can change it to simply print the `str(e)` if you think it is a better move.

Reply:
yea lets do that